.github/pull_request_template.md

```diff
@@ -13,3 +14,17 @@ Provide a concise overview of the issue or rationale behind the proposed changes
 - [ ] I have added or updated relevant tests.
 - [ ] I have added or updated relevant docs, including release notes if applicable.
 - [ ] I have attached screenshots in both light and dark modes for any UI changes.
+
+#### AI Assistance Disclosure (REQUIRED)
+Please select **exactly one** of the following:
+- [ ] **No AI tools were used** in preparing this pull request.
+- [ ] **AI tools were used**, and I confirm the following:
+
+<details>
+<summary>Required details for AI-assisted contributions</summary>
+
+- **Tool(s) used**: <!-- e.g., ChatGPT 4.0, GitHub Copilot, Gemini 1.5 Pro -->
+- **What it was used for**: <!-- e.g., drafting code, editing commit message, suggesting test cases -->
+- [ ] I have manually reviewed and verified all AI-generated content.
+- [ ] I confirm this PR adheres to Django's contribution and security policies.
```

Comment:
Suggestion: Should this point be moved out of the AI-specific section to a general point? I can't see a reason why this would only be applicable to AI-authored submissions?

Separately, should the template link out to said policies?

Reply:
yeap yeap yeap :100: